miden/tests/integration/stdlib/crypto/fri/mod.rs

```diff
@@ -30,14 +30,14 @@ fn fri_fold4_ext2_remainder32() {
     let depth = trace_len_e + blowup_exp;
     let domain_size = 1 << depth;
 
-    let (advice_provider, tape, position_eval, alphas, commitments, remainder, num_queries) =
+    let (advice_provider, stack, position_eval, alphas, commitments, remainder, num_queries) =
         fri_prove_verify_fold4_ext2(trace_len_e).expect("should not panic");
 
-    let tape = prepare_advice(
+    let stack = prepare_advice(
```

Comment:
should we use `advice_stack` rather than just `stack` here to make it more clear? (or maybe `adv_stack` for short?)